frontend/src/store/questions/constants.js

```diff
@@ -0,0 +1,23 @@
+import {
+  getQuestions,
+  getQuestion,
+  getAnswers
+} from '@/utils/api/questions'
+
+export const ACTIONS = {
+  GET_FEED: Symbol('getQuestionFeed'),
```

Comment:
idk what any of this is...
Also I think get_feed isn't descriptive enough; get_questions or get_question_feed is better

Reply:
yeap fair.